### PR TITLE
Added a fix for submission details page changes not committing new submissions

### PR DESCRIPTION
**Pull request by Mouli-Sanketh-Maturi**
This PR adds a fix that broke functionality due to leetcode website changes.

@arunbhardwaj Please let me know if you have any comments.

I tested the extension locally, and it works fine now.

Fixes Issue: https://github.com/arunbhardwaj/LeetHub-2.0/issues/12

### CODE REVIEW

File: scripts/leetcode.js [hunk 1]
```diff
@@ -81,46 +81,44 @@ const upload = (token, hook, code, problem, filename, sha, commitMsg, cb = undef
...
+      // create stats object
+      stats = {};
+      stats.solved = 0;
+      stats.easy = 0;
+      stats.medium = 0;
+      stats.hard = 0;
+      stats.shas = {};
+    }
 
-      if (stats.shas[problem] == null) {
```
Comment (arunachalamev) [inline scripts/leetcode.js:97]: Space are considered as a code change and its difficult to do Code review. Could you please resubmit without these changes

**Review by arunachalamev — CHANGES_REQUESTED** [review]

File: scripts/leetcode.js (hunk 1 above)
Comment (bjpfox) [reply scripts/leetcode.js:97]: Yes, perhaps try only making the change related to the issue #12 fix, but don't run npm prettier on the code

Comment (Mouli-Sanketh-Maturi) [reply scripts/leetcode.js]: @arunachalamev @bjpfox Reverted the prettier changes, please have a look.

Comment (bjpfox) [reply scripts/leetcode.js]: Looks good to me. I have also tested locally and can confirm everything works fine now. Great job 👍 

@arunachalamev I think this is good to merge, if you're happy to do so?

Comment (Prakhart-debug) [reply scripts/leetcode.js]: It is working for a new repository but not working for the repo which was used previously, so maybe some issues in linking

Comment (Mouli-Sanketh-Maturi) [reply scripts/leetcode.js]: @Prakhart-debug These changes doesn’t relate anything towards repository handling.

Can you share any console logs while commiting to the old repo?

Comment (bjpfox) [reply scripts/leetcode.js]: @Prakhart-debug I tested it using my previously used repo, and it worked perfectly.

After you installed / reloaded the chrome extension did you specify the same repo as previously used? for me, after I did that, everything worked.

**Comment by arunbhardwaj** [comment]
I find that this approach is more of a "forced submission" approach. I've detailed an alternative solution that I feel handles the new issue from the LeetCode UI changes in this PR:
https://github.com/arunbhardwaj/LeetHub-2.0/pull/18

I'd love to confirm that this works for you @Mouli-Sanketh-Maturi

**Pull request closed without merging**